saw-remote-api/src/SAWServer.hs

```diff
@@ -187,6 +186,8 @@ initialState readFile =
 #else
                 , roProxy = AIGProxy AIG.basicProxy
 #endif
+                , roInitWorkDir = cwd
+                , roBasicSS = undefined
```

Comment:
We probably shouldn't be using `undefined` anywhere. Do you have a plan for what should go here instead?

Reply:
FYI, for this line this has been working for me in my recent tinkering: https://github.com/GaloisInc/saw-script/commit/d4a6f8b5a855c647563dce25c7ab82787f90555f

Reply:
Oh, oops! I need to fill that in.